parser.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Text;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Linq;
+
+class day4
+{
+    public static string result;
+    //test for global Variable
+    public static void Main()
+    {
+        string url = @"https://www.microsoft.com/en-us/translator/business/languages/
+";
+
+        WebClient client = new WebClient();
+        client.Credentials = CredentialCache.DefaultCredentials;
+        byte[] data = client.DownloadData(url);
+        string html = Encoding.UTF8.GetString(data);
+        //Console.WriteLine(html);
+        //debug
+        List<row> textList = GetContent(html);
+        /*foreach (row line in textList) { 
```

Comment:
I prefer adding some checking:
1. If html is null or empty, and wrap that WebClient part in a try-catch or using(...){...}
2. If textList is empty, and then do something e.g. throw an exception, write to log, print in console...